### ISSUE TITLE
RadDataForm input fields look weird on iOS (like with double underline)

### ISSUE THREAD

**Issue by manoldonev** [NativeScript/theme#214]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 6.2.0-rc
 - Cross-platform modules: 6.2.0-rc
 - iOS Runtime: 6.2.0-rc
 - Plugin(s): nativescript-ui-dataform: 6.0.0-2019-10-30, @nativescript/theme: 2.2.0

**Describe the bug**
RadDataForm input fields look weird on iOS (like with double underline).

**To Reproduce**
Drag and drop a DataForm component in NS Playground.

**Expected behavior**
RadDataForm input fields' structure should not introduce double underline on iOS.

**Additional context**
![Screenshot 2019-10-30 at 18 28 21](https://user-images.githubusercontent.com/2650247/67877881-11b58480-fb43-11e9-9cac-c1e9da13d5ab.png)

**Comment by bundyo** [comment]
You're right - Dataform uses the standard for iOS form in a ListView, so the TextField underline should be removed to provide a more consistent look.
